src/main/java/com/petrych/screenshotter/common/FileUtil.java

```diff
@@ -30,4 +33,18 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
 		});
 	}
 	
+	public static String generateFileName() {
+		
+		int randomInt = new SecureRandom().nextInt();
+		String fileName = String.valueOf(randomInt);
+		String fileExtension = getFileExtension();
+		
+		return fileName.concat(fileExtension);
+	}
+	
```

Comment:
Do you really need all these variables here? Perhaps you can use String.format instead? For example, it seems unnessecarry to introduce `int randomInt` as it's only used once.